src/topology/subset_properties.lean

```diff
@@ -789,18 +803,108 @@ end
 variables (α) [sigma_compact_space α]
 open sigma_compact_space
 
-/-- An arbitrary compact covering of a σ-compact space. -/
+/-- A choice of compact covering for a σ-compact space, chosen to be monotone. -/
 def compact_covering : ℕ → set α :=
-classical.some exists_compact_covering
+accumulate exists_compact_covering.some
 
 lemma is_compact_compact_covering (n : ℕ) : is_compact (compact_covering α n) :=
-(classical.some_spec sigma_compact_space.exists_compact_covering).1 n
+compact_accumulate (classical.some_spec sigma_compact_space.exists_compact_covering).1 n
 
 lemma Union_compact_covering : (⋃ n, compact_covering α n) = univ :=
-(classical.some_spec sigma_compact_space.exists_compact_covering).2
+begin
+  rw [compact_covering, Union_accumulate],
+  exact (classical.some_spec sigma_compact_space.exists_compact_covering).2
+end
+
+@[mono] lemma compact_covering_subset ⦃m n : ℕ⦄ (h : m ≤ n) :
+  compact_covering α m ⊆ compact_covering α n :=
+monotone_accumulate h
 
 end compact
 
+/-- An [exhaustion by compact sets](https://en.wikipedia.org/wiki/Exhaustion_by_compact_sets) of a
+topological space is a sequence of compact sets `K n` such that `K n ⊆ interior (K (n + 1))` and
+`(⋃ n, K n) = univ`.
+
+If `X` is a locally compact sigma compact space, then `compact_exhaustion.some X` provides a choice
```

Comment:
```suggestion
If `X` is a locally compact sigma compact space, then `compact_exhaustion.choice X` provides a choice
```
(I don't have an opinion on whether it's better to use `some` or `choice`, but I think you use `choice` below.)